### ISSUE TITLE
Image alignment does not seem to work

### ISSUE THREAD

**Issue by RikSchefferAmsterdam** [gbumps/react-native-screenguard#67]
I cannot get the image alignment seem to work properly. I always get my image in the top left corner. With and without setting an alignment. This is my code:
```ts
        ScreenGuard.registerWithImage({
          backgroundColor: '#ff0000'
          defaultSource: require('./bootsplash_logo.png'),
          source: {
            uri: 'some uri',
          },
          height: 59,
          width: 167,
          // alignment: 4,
        })
```

I can get it positioned with using top and left, but that is not ideal:

```ts
        ScreenGuard.registerWithImage({
          backgroundColor: '#ff0000'
          defaultSource: require('./bootsplash_logo.png'),
          source: {
            uri: 'some uri',
          },
          height: 59,
          width: 167,
          top: (Dimensions.get('screen').height - 59) / 2,
          left: (Dimensions.get('screen').width - 167) / 2,
        })
```

I only tested on iOS

**Comment by gbumps** [comment]
on which react-native version, iOS, Android version ? Please be more specifics.

**Comment by RikSchefferAmsterdam** [comment]
react-native: 0.74.3
iOS: 17.5.1

**Comment by gbumps** [comment]
I'm closing and move to the newly created issue with the `registerWithImage`.  Please follows up #68

**Comment by gbumps** [comment]
reproduced on iOS 17.5.1, below not yet detected

**Comment by gbumps** [comment]
released and fixed on `1.0.2`. Please install and rebuild

Thanks!
